app/controllers/api/v1/rents_controller.rb

```diff
@@ -9,6 +9,8 @@ def index
       def create
         @rent = Rent.new(rent_params)
         @rent.save
+        @today = Time.zone.today.to_json
```

Comment:
I would do this inside the mailer method, also I think that you don't need to use the `.to_json`